content/blog/2020/08/2020-08-04-cdf-graduation.adoc

```diff
@@ -0,0 +1,166 @@
+Based on this review, we have made changes in our processes and documentation.
+It should improve experience of Jenkins users,
+and help to make the Jenkins community more welcoming to existing and newcomer contributors.
+
+== What changed for the project?
+
+Below you can find a few key changes we have applied during the graduation process:
+
+Public roadmap::
+
+We introduced a new link:/project/roadmap/[public roadmap] for the Jenkins project.
+This roadmap aggregates key initiatives in all community areas: features, infrastructure, documentation, community, etc.
+It makes the project more transparent to all Jenkins users and adopters,
+and at the same time helps potential contributors to find out the hot areas and opportunities for contribution.
+The roadmap is driven by the Jenkins community and it has a fully public process documented in jep:14[].
+
+More details about the public roadmap are coming next week, stay tuned for a separate blogpost.
+On July 10th we had an online contributor meetup about it,
+and you can find more information about the roadmap in its materials
+(link:https://docs.google.com/presentation/d/1_T2nZhP1WS2Fw0OLVAJV14Ke6nEsqBjLcdAHiygCmNs/edit?usp=sharing[slides], link:https://www.youtube.com/watch?v=ldWBY5BdQ5I[video recording]).
+
+User Documentation::
+
+* Jenkins Weekly Release line is now documented on our website (link:/download/weekly/[here]).
+  We have also reworked the downloads page and added guidelines explaining how to verify downloads.
+* A new link:/project/adopters[list of Jenkins adopters] was introduced on jenkins.io.
+  This list highlights Jenkins users and refers their case studies and success stories,
+  including ones submitted through the link:/blog/2020/04/30/jenkins-is-the-way/[Jenkins Is The Way] portal.
+  Please do not hesitate to add your companies there!
+
+Community::
+
+* We passed the link:https://www.coreinfrastructure.org/[Core Infrastructure Initiative (CII)] certification.
+  This certification helped us to verify compliance with open source best practices,
+  and to make adjustments in the project (see the bullets below).
+  It also provides Jenkins users and adopters with a public summary about compliance with each best practice.
+  See the Jenkins core's page link:https://bestpractices.coreinfrastructure.org/en/projects/3538[here].
+* link:https://www.jenkins.io/project/conduct/[Jenkins Code of Conduct was updated]
+  to the new version of Contributor Covenant.
+  In particular, it sets best practices of behavior in the community, and expands definitions of unacceptable behavior.
+* The default link:https://github.com/jenkinsci/.github/blob/master/CONTRIBUTING.md[Jenkins contributing template] was updated to cover more common cases for plugin contributors.
+  This page contributes to the link:/participate[Participate and Contribute] guidelines hosted on our website,
```

Comment:
Slightly different word so that "contribute" is not used twice in the same sentence?
```suggestion
  This page extends the link:/participate[Participate and Contribute] guidelines hosted on our website,
```

Reply:
```suggestion
  This page provides links to the link:/participate[Participate and Contribute] guidelines hosted on our website,
```